wazuh-dashboard/config/wazuh_app_config.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# Wazuh Docker Copyright (C) 2021 Wazuh Inc. (License GPLv2)
+
+wazuh_url="${WAZUH_API_URL:-https://wazuh}"
+wazuh_port="${API_PORT:-55000}"
+api_username="${API_USERNAME:-wazuh-wui}"
+api_password="${API_PASSWORD:-wazuh-wui}"
```

Comment:
These are the default username and password, if you assign the environment variables then it will configure the ones you put, otherwise you leave the default ones